src/modules/dashboard/components/ColonyDecisions/DetailsWidget/DetailsWidget.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
```

Comment:
I actually considered using the DetailsWidget you created [here]( https://github.com/JoinColony/colonyDapp/pull/3656), however this is not accessible to me as it has not been merged to master yet. I also considered using the DetailsWidget from the ActionsPage but it didn't seem like a good fit because it has a lot of things that are not relevant to decisions and it has a different style for displaying the user compared to what is needed for the decisions preview. Which DetailsWidget were you suggesting to use in this case?

Reply:
I think that you should use the DetailsWidget from the ActionsPage. This has the same UI. If the user is displayed differently, I would contact Arren/Karol to challenge that. I think that in the same scenarios users should be displayed the same - we don't need even more inconsistencies in the dapp. Karol told me that if there are any inconsistencies with the dapp we should use the UI from the dapp.
It doesn't matter that lots of things are not relevant - it is done specifically for this case. And we will be using it anyway for the decisions page once the transaction goes through so we'll be adding the same thing anyway. This way we can just reuse it.

Reply:
I confirmed with Arren and the design from the ActionsPage works for this. I actually had this originally built using the DetailsWidget from the ActionsPage but changed it to match the design but I will use the ActionsPage Detail widget now.